src/main/kotlin/uk/gov/justice/digital/hmpps/deliusapi/dto/v1/contact/SearchContactRequest.kt

```diff
@@ -60,6 +60,6 @@ data class SearchContactRequest(
   val sortDirection: SortDirection? = DESCENDING
 )
 
-enum class SortField(name: String) { RELEVANCE("relevance"), LAST_UPDATED_DATETIME("lastUpdated") }
+enum class SortField(name: String) { RELEVANCE("relevance"), LAST_UPDATED_DATETIME("lastUpdated"), MOST_RECENT_DATE_FIRST("mostRecentDateFirst") }
```

Comment:
Could we name this new field `CONTACT_DATE("contactDate")`?  Just to make a clearer distinction from the last updated date.

```suggestion
enum class SortField(name: String) { RELEVANCE("relevance"), LAST_UPDATED_DATETIME("lastUpdated"), MOST_RECENT_DATE_FIRST("mostRecentDateFirst") }
```